recipes/expect/build.sh

```diff
@@ -0,0 +1,12 @@
+aclocal
+autoconf
+( cd testsuite
+  autoconf -I.. )
+
+./configure --prefix=$PREFIX --build=$BUILD --host=$HOST --with-tclinclude=$PREFIX/include
+make -j ${CPU_COUNT}
+make test
+make -j ${CPU_COUNT} install
+
+mv $PREFIX/lib/tcl*/expect${PKG_VERSION}/libexpect${PKG_VERSION}.so $PREFIX/lib
```

Comment:
Does `make install` not do this for some reason? And are you sure that this move doesn't mess up any RPATHs?

Reply:
It's the side effect of a patch, and no it doesn't mess up RPATHs, it's actually a fix for a mess :)
xref: https://src.fedoraproject.org/rpms/expect/blob/f33/f/expect.spec#_123